src/routes/(docs)/Toast/ExampleToast.svelte

```diff
@@ -0,0 +1,20 @@
+<script>
+	import { Typography, ToastButton, Toast } from '$lib';
+</script>
+
+<ToastButton timeout={3000}>Click me</ToastButton>
+
+<Toast variant="warning">
+	<Typography variant="sub-heading">Invalid Username</Typography>
+	<Typography variant="body">Username you have entered is invalid</Typography>
+</Toast>
```

Comment:
I'm not totally sure how to practically use a Toast with this API. Say you have just fired off a request to save some data. You want to show a toast once that has saved or errored. The message text may be dynamic based on the response of the request. You want to be able to fire a call off something like:
```js
showToast({
  variant: "warning",
  heading: "Invalid Username",
  message: "Username you have entered is invalid",
  timeout: 5000
});
```

then that gets rendered in a toast area in some corner of the screen. You may also end up with multiple toasts shown at the same time.